wh.order.import.cached.js

```diff
@@ -276,10 +278,53 @@ try {
                                       {concurrency: 1}
                                     )
                                       .then(function(){
```

Comment:
you could blackbox this into a file ... like the logic is compressed earlier in:
`var prepStockOrder = require('./jobs/cache-vend-products-for-stock-order.js');`